plugins/keycloak_oidc_login/messages/de/keycloak_oidc_login.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'login' => 'Mit Keycloak anmelden',
```

Comment:
I would prefer if it would be possible to parametrize the name of the login provider. Either in config file or in the admin backend.

Reply:
All messages defined here can be overwritten using the Web UI anyway. It should pop up as a separate tab in the language message configuration

Reply:
ok perfect.